### PR TITLE
Easyconfig for QuantumESPRESSO-5.4.0 with FOSS of 2016, hybrid version. 

### PR DESCRIPTION
**Pull request by LiangFr**
OpenMP-MPIhybrid easyconfig should be used with the corresponding easyblock in the link [Easyblock pull request 954](https://github.com/hpcugent/easybuild-easyblocks/pull/954) @aocais

### CODE REVIEW

**Comment by ocaisa** [comment]
This requires https://github.com/hpcugent/easybuild-easyblocks/pull/954

File: easybuild/easyconfigs/q/QuantumESPRESSO/QuantumESPRESSO-5.4.0-foss-2016-hybrid.eb [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+version = '5.4.0'
+
+homepage = 'http://www.pwscf.org/'
+description = """Quantum ESPRESSO  is an integrated suite of computer codes
+ for electronic-structure calculations and materials modeling at the nanoscale.
+ It is based on density-functional theory, plane waves, and pseudopotentials
+  (both norm-conserving and ultrasoft)."""
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+
```
Comment (ocaisa) [inline easybuild/easyconfigs/q/QuantumESPRESSO/QuantumESPRESSO-5.4.0-foss-2016-hybrid.eb]: Delete this line

File: easybuild/easyconfigs/q/QuantumESPRESSO/QuantumESPRESSO-5.4.0-foss-2016-hybrid.eb [hunk 2]
```diff
@@ -0,0 +1,60 @@
+name = 'QuantumESPRESSO'
+version = '5.4.0'
+
```
Comment (ocaisa) [inline easybuild/easyconfigs/q/QuantumESPRESSO/QuantumESPRESSO-5.4.0-foss-2016-hybrid.eb]: Add 
```
versionsuffix = '-hybrid'
```
so the filename matches correctly

File: easybuild/easyconfigs/q/QuantumESPRESSO/QuantumESPRESSO-5.4.0-foss-2016-hybrid.eb [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    'http://www.qe-forge.org/gf/download/frsrelease/211/958/', # xspectra-5.4.0.tar.gz
+    'http://www.qe-forge.org/gf/download/frsrelease/211/969/', # EPW-5.4.0.tar.gz
+    'http://www.qe-forge.org/gf/download/frsrelease/211/963/', # test-suite-5.4.0.tar.gz
+    'http://www.qe-forge.org/gf/download/frsrelease/211/957/', # GWW-5.4.0.tar.gz
+]
+
+# Hybrid option exist only for backward compatibility, use toolchain option openmp instead.
+#hybrid = True
+  
+#Build QE with packages of Pwscf, tddfpt, NEB, xspectra, Phonon of  QE 5.4.0. While yambo and someother packages can not be successfully built with this easyconfig with QE 5.4.0 version!
```
Comment (ocaisa) [inline easybuild/easyconfigs/q/QuantumESPRESSO/QuantumESPRESSO-5.4.0-foss-2016-hybrid.eb]: 120 character limit, please split this comment

**Comment by ocaisa** [comment]
Didn't some systems (JURECA) require a patch on `make.sys` for the pre-processor? The QE build system seems to do different things on different OSes.

**Comment by boegel** [comment]
@ocaisa I presume you're in touch with @LiangFr over this? Why was the PR closed?

**Comment by ocaisa** [comment]
Yes, he's going to open a new one from a different branch with the patch...still trying to get the workflow right ;)

**Comment by boegel** [comment]
:+1:

**Pull request closed without merging**